src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1526,6 +1527,14 @@ public String toVariableNameString(EnchantmentOffer eo) {
 				.name("Transform Reason")
 				.description("Represents a transform reason of an <a href='events.html#entity transform'>entity transform event</a>.")
 				.since("2.8.0"));
+
+		if (Skript.classExists("org.bukkit.inventory.ItemFlag")) {
```

Comment:
This class has been in Bukkit since the dawn of time, so I dont think you need a check here.

Reply:
ah, i didnt know that so i thought i should put it there just to be safe, thanks